app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split(" ")
+
+    if "cp" not in files and files.index("cp") != 0:
+        print("Incorrect command to copy file")
+        return
+
+    if files[1] == files[2]:
+        return None
```

Comment:
if a function does not have an explicit return statement, it returns None by default